### ISSUE TITLE
Add "render props" component to React cheatsheet

### ISSUE THREAD

**Issue by DalerAsrorov** [leon-good-life/developer-cheatsheets#1]
Thanks a lot for the cheatsheet! Would you mind if we add the "render props" example with component?

https://reactjs.org/docs/render-props.html

**Comment by leon-good-life** [comment]
Hi @DalerAsrorov ,

Thanks for your comment.

Sure, you can make a PR and add an example, you will receive a credit under the example with a link to your website.

To add an example to an React Cheatsheet:
1. create a directory inside 'react' directory.
2. Inside the directory you created:
2.1. create content.txt file and write there the example.
2.2. create metadat.json file

{
	"title": "WRITE HERE THE NAME OF THE EXAMPLE",
	"file": "content.txt",
	"type": "language-jsx",
        "author": "WRITE HERE YOUR NAME"
}

Thanks a lot,

Kind regards,
Leon.

**Comment by leon-good-life** [comment]
Thanks @DalerAsrorov for the PR, I merged it, 
And it is now live on the website.

If you don't mind I'm making minor changes to your code:
- Semicolons: in some places there are semicolons and in some parts there are no semicolons, I removed the semicolons, so it is now more consistent.
- Long lines in mobile view: To make the website look good on mobile devices I will make the lines short. (I tried also, in my code, to add css media query and decrease the font-size on mobile).
